README.md

```diff
@@ -1,2 +1,10 @@
-# auth-server
-Authentication
+# AUTH-SERVER
+# Authentication
+
```

Comment:
The template for the readme can be found here https://github.com/codefellows/seattle-javascript-401n17/blob/master/reference/submission-instructions/labs/README-template.md.